docs/product/performance/web-vitals/index.mdx

```diff
@@ -60,6 +60,10 @@ The **Page Overview** page displays a "Page Loads" chart in the right sidebar of
 
 ### Samples List
 
+<Note>
+The samples list only shows page loads with web vitals and performance score. If a page load is missing a required web vital, it will not be shown in the samples list.
```

Comment:
Can we add reference to the Browser Support section to help them understand why a pageload may not capture web vitals?

Reply:
Updated!